docs/reference/src/language/widgets/verticalbox.md

```diff
@@ -0,0 +1,4 @@
+## `VerticalBox`
+
+A `VerticalBox` is a [`VerticalLayout`](../builtins/elements.md#verticallayout-and-horizontallayout) where the spacing and padding values
+depend on the style instead of defaulting to 0.
```

Comment:
(I know that changing the content is not for this PR which is only about structure, but this page is a bit strange.  I wonder if it could be merged with HorizontalBox. But since they are both actual "components" they probably deserve their own page by consistancy.  
What's for sure is that they could link to each other with "see also".

Reply:
I agree - the pages are quite weird for being so empty. I'll merge them into one in a follow-up.